src/mirror.c

```diff
@@ -208,6 +232,13 @@ static int parse_options(int argc, char **argv)
 			}
 			unset = true;
 			break;
+		case 'F':
```

Comment:
Why in the world is this necessary for setting a mirror URL?

Reply:
Without -F staging, swupd mirror fails in the test environment.

Reply:
Then let's fix the test environment instead of adding a useless command to the interface.

Reply:
In your test setup do this:
```bash
sudo mkdir -p "$DIR/target-dir/usr/share/defaults/swupd"
echo 1 | sudo tee "$DIR/target-dir/usr/share/defaults/swupd/format"
```

Reply:
Do I just need to throw an arbitrary positive integer into "$DIR/target-dir/usr/share/defaults/swupd/format"?

Reply:
yes